backend/app/endpoints.py

```diff
@@ -54,10 +55,13 @@ def from_treasury_with_assets(
         assets = {
             a.token_symbol: PortfolioAsset(
                 allocation=a.balance / treasury.usd_total,
-                volatility=histprices[a.token_symbol]["std_dev"].mean(),
+                volatility=None
+                if histprices[a.token_symbol]["std_dev"].mean() is NaN
```

Comment:
Less of a problem here than the one mentioned below. But still: what does it mean if there is a NaN? Is it something that we need to let happen (because of missing quote data from the data source), or can we rather fill in the gaps so as to make sure that the std dev is always there, even if approx?

Reply:
This error (of `histprices[a.token_symbol]["std_dev"].mean()` being `NaN`) only arises if the `len(histprices[a.token_symbol]["std_dev"])` is 0.

So this error is only encountered if `histprices[a.token_symbol]` is empty;  When the timeframes were <= 3 months for OWL.

Therefore maybe it's worth keeping this? If `len(histprices[a.token_symbol]) == 0` then the asset does not have any historical price data from coingecko. At this point, we don't have a way to mitigate this with a fallback and this PR was not created to address this issue.

Reply:
Ok.